frontend/pages/login.tsx

```diff
@@ -71,7 +80,7 @@ export default function Login() {
           const data = await res.json();
           await mutateUser({
             isLoggedIn: true,
-            roles: data.roles,
+            role: data.roles[0],
```

Comment:
Besch secher das mer nor das Attribute 0 wöndt notze vode Rollene? ech glaub mer bechömit doch en Array full of Enums, oder?

Reply:
ja. wir hend im refinement bestimmt, dass 1 user nur 1 rolle ha chan. daher hanis ahpasst